src/components/Form.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+class Form extends React.Component {
+  render() {
+    return (
+      <form onSubmit={this.props.getWeather}>
```

Comment:
Very good! You're passing the `getWeather` action up to the parent to make the API request 💪